src/mono/mono.proj

```diff
@@ -464,11 +464,21 @@
     <!-- Linux options -->
     <ItemGroup Condition="'$(TargetsLinux)' == true">
       <_MonoCFLAGS Include="-Wl,--build-id=sha1" />
+      <_MonoCFLAGS Include="-Wno-int-conversion" />
       <_MonoCXXFLAGS Include="-Wl,--build-id=sha1" />
+      <_MonoCXXFLAGS Include="-Wno-int-conversion" />
     </ItemGroup>
     <ItemGroup Condition="'$(RealTargetOS)' == 'Linux'">
       <_MonoAOTCFLAGS Include="-Wl,--build-id=sha1" />
+      <_MonoAOTCFLAGS Include="-Wno-int-conversion" />
       <_MonoAOTCXXFLAGS Include="-Wl,--build-id=sha1" />
+      <_MonoAOTCXXFLAGS Include="-Wno-int-conversion" />
```

Comment:
Are these CXXFLAGS needed as well? Mono is mainly a C project with a few C++ files compiled when `mono.mscordbi`  subset is built (it is optional and a small component). Mirroring every C flag to C++ is not needed.

Reply:
Ideally, I'm looking to remove the necessity for `-Wno-int-conversion`. I'll try removing the CXX flags and see how that goes.